### ISSUE TITLE
Add metadata that maps individuals to roles, and use that metadata to generate text.

### ISSUE THREAD

**Issue by melodykramer** [18F/hub#402]
Goal: To not have to update every page on the Hub every time a person leaves

Idea: Keep metadata that maps individuals to roles, and use that metadata to generate text.

Capturing conversation from Slack: 

So something like this:
```walrus:
  name: mbland
  full_name: Mike Bland
```
And then the template would actually have: `[{{ site.data.contacts.walrus.full_name }}]({{ site.baseurl }}/team/{{ site.data.contacts.walrus.name }})`

We could make it easier by creating a plugin that’ll expand the entry into a link automatically, so you’d just have to write `{% contact_link walrus %}`.

**Comment by kategarklavs** [comment]
I love this idea. Along similar lines, @emileighoutlaw and I found, in the course of our wg-content research, that lots of folks aren't familiar with the content resources that are available. We'd love to be able to create a content "directory" of sorts — a searchable directory of content folk that lists each person's skills/preferences/aptitudes/current projects. Seems like this might intersect nicely with the current tagging initiative ... 

cc @melodykramer @mbland

**Comment by mbland** [comment]
@kategarklavs Something like this?

https://hub.18f.gov/skills/
https://hub.18f.gov/interests/

Except perhaps where the skills and interests themselves are categorized?

**Comment by mbland** [comment]
Oh, also, I'd built in the concept of "departments" early on, but it just hasn't gotten much attention:

https://hub.18f.gov/departments/

They're also still in the [Team API](https://team-api.18f.gov/public/api/). Happy to start fleshing out those concepts, either here or in [18F/team_api](https://github.com/18F/team_api).

**Comment by kategarklavs** [comment]
@mbland Yes! That looks great (and I'd be happy to help out, in what ways I can, with the fleshing out of these concepts — let me know if you need content assistance).

**Comment by emileighoutlaw** [comment]
I think this would be a great place to put content!

**Comment by melodykramer** [comment]
A perfect example for why we need this: https://github.com/18F/hub/blob/master/_pages/qa/it-service.md

**Comment by ultrasaurus** [comment]
@dhcole did a bunch of integration work this spring with Midas... is there any way we can leverage that?  The idea was that we could use bio, title, skills and interests from the Midas profile since it is easy for people to edit there.

**Comment by mbland** [comment]
Yes, I've been meaning to import Midas data into the Team API. Just haven't gotten around to it yet.

**Comment by melodykramer** [comment]
@mbland Do you know how far out on the roadmap this is? We're designing pages for the Hub that could really use this.

**Comment by mbland** [comment]
We can start getting it together now. Create a `_data/roles.yml` file that could look something like this:

```yaml
ops_contact: batemapf
talent_contact: jamie
onboarding_contact: melody
```

And so on. Then in the pages, we can start using `{{ site.data.roles.onboarding_contact }}` and so on. When the day comes we switch this over to importing from the Team API, the pages themselves shouldn't need any updates.

Playing around with the local data file in this way may also help us try out a few ideas before committing it to the Team API.

**Comment by nicoleslaw** [comment]
@mbland @mtorres253 Is this still in progress?

**Comment by mbland** [comment]
In the sense that we haven't actually started working on it, yes, still in progress. :smiley: I'd imagine it would be a relatively small task to add it to the Team API, then perhaps a slightly larger task to integrate it into the Handbook. We should scope and prioritize it at the next planning meeting.
